src/styles/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  border: 1px solid var(--gray-elements);
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+  background-color: var(--white);
```

Comment:
Ensure that the SCSS variable `--white` is defined in a separate variables file and imported correctly.